tests/logprob/test_binary.py

```diff
@@ -0,0 +1,70 @@
+#   Copyright 2023 The PyMC Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+import numpy as np
```

Comment:
I think you need to add this file here: https://github.com/pymc-devs/pymc/blob/5d68bf3063c999b3d9bb2ec0678e3aa32611baaa/.github/workflows/tests.yml#L106